media-video/obs-studio/metadata.xml

```diff
@@ -17,8 +17,10 @@
     <flag name="luajit">Build support for scripting via Luajit.</flag>
     <flag name="nvenc">Enable NVENC encoding for nVidia video cards.</flag>
     <flag name="python">Build support for scripting via Python 3.</flag>
+    <flag name="rtmps">Build support for RTMPS.</flag>
     <flag name="speex">Enable noise suppression filter support via
       <pkg>media-libs/speexdsp</pkg>.</flag>
+    <flag name="vlc">Build support for VLC.</flag>
```

Comment:
Does it *use* `media-video/vlc` for something or does it add something for VLC?

Reply:
Again not sure why I left it as plain as I did.  Will fix.